### ISSUE TITLE
docs_path and ignore_headers options not having any effect

### ISSUE THREAD

**Issue by DavidOliver** [rhazdon/blue_bird#15]
This is the config in my umbrella app's child app:

```
config :blue_bird,
  docs_path: "priv/static/docs2",
  theme: "triple",
  router: MyApp.Web.Router,
  ignore_headers: ["cache-control", "x-request-id"]
```

The router config is picked up (without it I get `module MyApp.Application.Router is not available`), but the `docs_path` and `ignore_headers` don't seem to have any effect.

- The apib file is output to `priv/static/docs` (relative to the parent umbrella project rather than the child app) regardless of the value I use.
- The response in the apib file includes `cache-control` and `x-request-id` headers.

Edit: just a further note/question on the `docs_path`: should it be relative to an umbrella app's root or to the child app's directory?

**Comment by woylie** [comment]
`docs_path` is currently relative to the umbrella app's root.

Just to be clear, the configuration above is in `apps/myapp_web/config/config.exs`, right?

**Comment by woylie** [comment]
And just to be sure, did you restart your app after changing your config?

**Comment by DavidOliver** [comment]
> the configuration above is in apps/myapp_web/config/config.exs, right?

It's in `my_app_umbrella/apps/my_app/config/config.exs`, yes. (I don't think it's relevant, but it's not in an app named `..._web` as my child app handles both its own "main" and "web" concerns, having its own `web` subdirectory in `lib`.) I've also tried putting the config in `my_app_umbrella/config/config.exs`.

> did you restart your app after changing your config?

My app isn't running apart from when I do `mix test` (or `mix phx.server`) as I'm only in dev so far. I did recompile with `mix compile` just in case that made a difference, but it didn't.

**Comment by woylie** [comment]
I'll have to investigate this further, because we're using BlueBird in an umbrella app without any configuration issues.

**Comment by DavidOliver** [comment]
Okay. When I do `mix bird.gen.docs`, the `docs_path` value _is_ used and I get a ENOENT error as it tries to load the apib files from `docs2`, but it's actually in `docs`. If I change the setting back to `docs` it loads the file okay.

**Comment by woylie** [comment]
Hi @DavidOliver,

I can't reproduce the issue, but I attempted a fix anyway. Could you please checkout the branch [fix/path-and-ignore-header-config](https://github.com/KittyHeaven/blue_bird/tree/fix/path-and-ignore-header-config) and check whether the problem persists?

**Comment by DavidOliver** [comment]
Hi @woylie. That seems to have fixed it. The apib file is now put in `priv/static/docs2` when I specify that path in the config. Many thanks.

Edit: and the headers set to be ignored are no longer present.

**Comment by woylie** [comment]
fixed in PR #19
